docs/100-upgrade/migration-guides/3.7-3.8.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: 3.7 -> 3.8
+description:
+  This page lists the highlights for upgrading a project from Front-Commerce 3.7
+  to 3.8
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<p>{frontMatter.description}</p>
+
+## Update dependencies
+
+Update all your `@front-commerce/*` dependencies to this version:
+
+```shell
+pnpm update "@front-commerce/*@3.8.0"
+```
+
+## Automated Migration
+
+We provide a codemod to automatically update your codebase to the latest version
+of Front-Commerce. This tool will update your code when possible and flag the
+places where you need to manually update your code (with `// TODO Codemod`
+comments).
+
+```shell
+pnpm run front-commerce migrate --transform 3.8.0
+```
+
+## Additional SEO meta data
+
+### Category Schema
+
+See
+[full diff](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/3658/diffs)
+for more details.
+
+#### Fields
+
+- [`meta_robots`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/173deec49216145c45a703c6f3353fc7dc631d26/packages/core/graphql-core-schema/category/index.ts#L58) -
+  A string containing the `robots` meta tag content.
+
+:::note
+
+We added a new
+[`generateCategorySeoMetas`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/173deec49216145c45a703c6f3353fc7dc631d26/packages/theme-chocolatine/theme/modules/Category/CategorySeo/generateCategorySeoMetas.ts)
+function in `theme/modules/Category/CategorySeo` to extract the logic used in
+the
+[`routes_main.category.$id.tsx`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/173deec49216145c45a703c6f3353fc7dc631d26/packages/theme-chocolatine/routes/_main.category.$id.tsx#L119)
+route.
```

Comment:
**nitpick:** in a migration guide, we should focus on what **integrators** needs to do, more than on what **we** did

**suggestion:** 
```suggestion
We added a new
[`generateCategorySeoMetas`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/173deec49216145c45a703c6f3353fc7dc631d26/packages/theme-chocolatine/theme/modules/Category/CategorySeo/generateCategorySeoMetas.ts)
function in `theme/modules/Category/CategorySeo` to extract the logic used in
the
[`routes_main.category.$id.tsx`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/173deec49216145c45a703c6f3353fc7dc631d26/packages/theme-chocolatine/routes/_main.category.$id.tsx#L119)
route.

If you overrode the `routes/main.category.$id.tsx` route, we recommend that you update it accordingly.
```